### DISCUSSION TITLE
Output images folder?

### DISCUSSION THREAD

**Discussion by yanlec** [Acly/krita-ai-diffusion#933] [Q&A] [answered]
Where are the generated images stored? I am assuming there is an output folder somewhere since generating images doesn't increase the krita file. I looked couple places on m system but I can't find it, and the settings window for the plugin doesn't show where they are saved.

Thanks

**Comment by Acly** [comment, marked as answer]
They're inside the .kra file (compressed, but they do take some space)

**Reply by yanlec** [reply]
Ah ok got it! Thank you!
